### ISSUE TITLE
Google Play returns "Varies by Device"

### ISSUE THREAD

**Issue by KennyGoers** [javiersantos/AppUpdater#13]
Compare of versions crashes when Google Play store has old versions for old devices...

**Comment by javiersantos** [comment]
Do you have an example app where I can reproduce the issue?

Thanks.

**Comment by KennyGoers** [comment]
com.trixiesoft.clapp on the play store

**Comment by javiersantos** [comment]
Fixed in the latest version of the library (v1.2.1). Unfortunately, apps showing "Varies by Device" on Google Play are not supported by the library. Thanks for letting me know!

```Javascript
dependencies {
    compile 'com.github.javiersantos:AppUpdater:1.2.1'
}
```
